konfera/migrations/0004_auto_20161012_1937.py

```diff
@@ -8,7 +8,7 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('konfera', '0003_auto_20161011_1907'),
+        ('konfera', '0002_schedule_event.py'),
```

Comment:
@sourabh1024 the extension `.py` shouldn't be there. It should be just `('konfera', '0002_schedule_event'),`.

Try to change it.